### ISSUE TITLE
lightbox not working on mobile devices

### ISSUE THREAD

**Issue by janosfarkas** [MurhafSousli/ngx-gallery#51]
```
- [X] bug report
- [ ] feature request
- [ ] question
```

### OS and Version?
iOS 11.2.2/Android 8.0/Android 6.0.1

### Versions
- online demo: https://murhafsousli.github.io/ng-gallery/#/lightbox
+on my localhost:
Angular CLI: 1.6.3
Node: 8.9.0
Angular: 5.1.3

### Repro steps
-Online demo- Lightbox simply not working on mobile devices(cant open images in lightbox, screen turns darker, but not showing anything at all)
-localhost 
-- iOS - on first try opens the images, but after closing and reopening the image is black, but after swiping on the next image and back displays correctly
-- Android 8.0/6.0.1 (Samsung galaxy S7 and Galaxy Note - freezes often, huge delay between swiping, and clicking on close button when other button is "behind" on UI it calls both events.

**Comment by MurhafSousli** [comment]
Hi @janosfarkas the lightbox is actually working, but it needs some CSS adjustments, try landscape mode and it will appear.
to fix this just add the following in `style.scss`
```css
.g-overlay {
   min-width: 360px;
}
```
try https://ng-gallery.stackblitz.io/ on your phone 

The demo photos are very large I guess each photo is around 5-7MB, that's why it is causing some issues, I am currently working on a new version fix these issues

**Comment by MurhafSousli** [comment]
fixed in version 2 #58
